crates/ar-cli/src/generate.rs

```diff
@@ -35,10 +39,17 @@ pub fn run(seed: &[u8; 32], dir: PathBuf, endianess: cli::Endianess) -> Result<(
 
     let (private_key, public_key) = generate_keys(&mut rng);
 
-    debug!("private key: : {private_key:?}");
+    debug!(
+        "private key: : {private_key:?} [{}]",
+        field_to_u256(private_key)
+    );
```

Comment:
do we want to print this to console? (some security concerns)

Reply:
added a redact